backend/app/repositories/user.py

```
@@ -0,0 +1,63 @@
+    UserCreate,
+    UserUpdate,
+    get_password_hash,
+    verify_password,
+)
+from app.repositories.base import BaseRepository
+
+
+class UserRepository(BaseRepository[User, UserCreate, UserUpdate]):
+    """User repository for database operations."""
+
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)